src/py/flwr/cli/new/templates/app/code/flwr_tune/strategy.py.tpl

```diff
@@ -0,0 +1,57 @@
+"""$project_name: A Flower / FlowerTune app."""
+
+from logging import INFO, WARN
+from typing import List, Tuple, Union
+
+from flwr.server.strategy import FedAvg
+from flwr.server.client_proxy import ClientProxy
+from flwr.common import (
+    log,
+    FitRes,
+    Parameters,
+    parameters_to_ndarrays,
+)
+
+
+class FlowerTuneLlm(FedAvg):
+    """Customised FedAvg strategy implementation."""
+    def configure_fit(self, parameters: Parameters, **kwargs):
+        """Configure the next round of training."""
+        return_clients = super().configure_fit(parameters=parameters, **kwargs)
+
+        # Test communication costs
+        num_clients = len(return_clients)
+        test_communication_costs(parameters, num_clients)
+
+        return return_clients
+
+    def aggregate_fit(
+        self,
+        server_round: int,
+        results: List[Tuple[ClientProxy, FitRes]],
+        failures: List[Union[Tuple[ClientProxy, FitRes], BaseException]],
+    ):
+        """Aggregate fit results using weighted average."""
+        # Test communication costs
+        num_clients = len(results)
+        test_communication_costs(results[0][1].parameters, num_clients)
+
+        parameters_aggregated, metrics_aggregated = super().aggregate_fit(server_round, results, failures)
+
+        return parameters_aggregated, metrics_aggregated
+
+
+def test_communication_costs(parameters, num_clients):
```

Comment:
I think it would be better if this function receives `Union[List[FitIns],List[FitRes]]` and use that to count the parameters communicated. The current implementation assumes the same size being sent/received to/from all clients.